config.yaml.defaults

```diff
@@ -102,3 +104,11 @@ cron:
   - interval: 1440
     script: jobs/delete_unsaved_candidates.py
     limit: ["01:00", "02:00"]
+
+invitations:
+  # Note that in order for invitations to be sent, a valid Sendgrid API key must be
+  # available as an environment variable as per their setup docs
+  days_until_expiry: 3
+  email_subject: "You've been invited to collaborate on SkyPortal"
```

Comment:
@dmitryduev not a request just a comment. this is what must be overloaded in fritz